AirGradient-ESPHome/README.md

```diff
@@ -14,12 +14,13 @@ First, create a `secrets.yaml` file in this directory with the following content
 name: airgradient-office
 
 # Set passwords for HA API and OTA updates.
-api_password: apipassword
-ota_password: otapassword
+api_password: airgradientjjg
+ota_password: airgradientjjg
 
 # Configure a WiFi network connection.
-wifi_ssid: my-wifi-network
-wifi_password: my-wifi-password
+wifi_ssid: geerling
+wifi_password: mineralocorticoids
```

Comment:
Heh I leaked my test network password here... can you make sure this is not in the commit? Also make sure nothing else here is changed, since the .ino file should contain the changes.

Reply:
Hey, sorry I missed that. Is it now sufficient?